src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Converters/Object/ObjectConverterFactory.cs

```diff
@@ -113,8 +113,6 @@ private JsonConverter CreateKeyValuePairConverter(Type type, JsonSerializerOptio
                 args: null,
                 culture: null)!;
 
-            converter.Initialize(options);
```

Comment:
Similarly, can you check if `Initialize` is used elsewehere and delete if dead code?

Reply:
It is used in a few places.